hello_world/src/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include(ExternalProject)
 include(GNUInstallDirs)
 
 set_target_properties(${PROJECT_NAME} PROPERTIES PUBLIC_HEADER
-        ${CMAKE_CURRENT_SOURCE_DIR}/include/hello-world.hpp
+        ../include/hello-world.hpp
```

Comment:
You should always use absolute paths in CMake files. Also why did you move this file?

Reply:
I am not sure why it moved let me check and fix that.